theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
+
+header p
+{
+  display: inline;
+  opacity: 0.7;
+  color: #000000;
+  padding: 0 10px;
+}
+
+header img
+{
+ vertical-align: middle;
+ width: 32px;
+ height: auto;
+}
+
+#hamburger
+{
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  background-image: url(/images/hamburger.svg);
+  width: 32px;
+  height: auto;
+}
+
+@media (max-width: @tablet-width)
+{
+  header
```

Comment:
Not sure if this style intended to apply only to mobile?

Reply:
This still relevant and you seems like still duplicating this style both in: `header nav` and here. 

It's just about header's bottom shadow, isn't it? This suppose to be declared in 1 place and apply both to mobile and desktop, unless I'm missing something.

Reply:
You're right. Sorry, I meant header `nav changes`, my fault. I'll remove the `header nav` shadow and there will be only one property on `header`.

Reply:
Done.